src/redux/initiative-tracker/initiativeTrackerSlice.ts

```diff
@@ -91,11 +91,19 @@ export const initiativeTrackerSlice = createSlice({
       const index = adjustedCreatureIndex(state, action.payload.index);
       state.creatures[index].initiative = newInitiative === "" ? undefined : Number(newInitiative);
       sortInitiativeCreatures(state.creatures);
+    },
+    rollAllInitiative: (state) => {
+      state.creatures.forEach((creature) => {
+        const min = Math.ceil(1);
+        const max = Math.floor(20);
```

Comment:
`min` and `max` are always 1 and 20, respectively... `Math` functions are not needed.